code/__HELPERS/roundend.dm

```diff
@@ -435,6 +460,21 @@
 		return "<div class='panel stationborder'>[parts.Join("<br>")]</div>"
 	return ""
 
+///Generate a report for all players who made it out alive with a hardcore random character and prints their final score
+/datum/controller/subsystem/ticker/proc/hardcore_random_report()
+	var/list/parts = list()
+	parts += "<span class='header'>The following people made it out as a random hardcore character:</span>"
+	parts += "<ul class='playerlist'>"
+	for(var/i in GLOB.human_list)
```

Comment:
I'd check `GLOB.player_list` instead and check for being human and alive.
Way less mobs on that list, and it would avoid someone getting the message because someone dragged their body to shuttle.